include/bluetooth/direction.h

```diff
@@ -76,8 +85,12 @@ struct bt_df_adv_cte_tx_param {
  * for correctness.
  */
 struct bt_df_per_adv_sync_cte_rx_param {
-	/* Bitmap with allowed CTE types (@ref bt_df_cte_type). */
-	uint8_t cte_type;
+	/**
+	 * @brief Bitfield with allowed CTE types.
+	 *
+	 *  Allowed values are defined by @ref bt_df_cte_type, except BT_DF_CTE_TYPE_NONE.
```

Comment:
Is there a usecase for `BT_DF_CTE_TYPE_NONE`?

Reply:
It was added purposely in the past, but it looks that it is not necessary anymore. 
I'll create a separate PR to remove it.